base/sysinfo.jl

```diff
@@ -14,10 +14,63 @@ export CPU_CORES,
        uptime,
        loadavg,
        free_memory,
-       total_memory
+       total_memory,
+       isapple,
+       isbsd,
+       islinux,
+       isunix,
+       iswindows
 
 import ..Base: show
 
+"""
+    isunix([os])
```

Comment:
I didn't because I thought there was another place where we don't but I can't think of an example so I might be making that up. I'll add them.